app/models/topic.py

```diff
@@ -54,6 +54,16 @@ async def get_all() -> ['TopicModel']:
             response = await table.scan()
         return sorted(list(map(TopicModel.from_dict, response['Items'])), key=lambda topic: topic.slug)
 
+    @staticmethod
+    async def get_topics(topics_ids: Set[str]) -> ['TopicModel']:
+        """
+        Retrieves all topics from dynamo db
+
+        :return: a list of TopicModel objects
+        """
+        all_topics = await TopicModel.get_all()
```

Comment:
It looks like you switch to Active Record style models here. Why a different strategy than above with user recommendation preferences?